### PR TITLE
integrated notes from guild leaders meeting

### PR DESCRIPTION
**Pull request by ultrasaurus**
The guild leaders met a few weeks ago with @quepol and @mbland.  This is my attempt at summarizing the results of the decisions at that meeting as modification to our process.

### CODE REVIEW

**Comment by ultrasaurus** [comment]
@quepol pls review

**Comment by ultrasaurus** [comment]
Also @gbinal @msecret @khandelwal @dlapiduz @nickbristow @afeld @meiqimichelle @konklone @rjmajma @yozlet

**Comment by meiqimichelle** [comment]
Looks good to me! Thank you for writing/summarizing, @ultrasaurus !

File: pages/guilds.md [hunk 1]
```diff
@@ -2,42 +2,44 @@
...
+including taking steps to measure progress.  The guild may establish
+requirements for project teams; however, those requirements must clearly
+contribute to project success or legal requirements, and the guild must support
+the project team in meeting those requirements.
+
+
+Guild leaders have a responsibility to make sure we, 18F, are strong in the
+guild area of expertise.  
+If we are, then the guild can provide passive resources and be responsible
+for periodic evaluation or other measure to monitor the strength of the team.
```
Comment (mbland) [inline pages/guilds.md]: Nit: "measure" => "measures".

File: pages/guilds.md [hunk 2]
```diff
@@ -2,42 +2,44 @@
...
+for periodic evaluation or other measure to monitor the strength of the team.  
+If needed, guild leaders must carve out enough time to ensure that the guild
+provide appropriate resources for the team, which may involve requesting additional people
+to devote dedicated time for specific projects or activities that are needed to support the work of the guild.
 
 ## <a name="formation"></a>Formation
 
 Guilds can come into existence via one of two avenues: Leadership can request
-that a team member lead a particular guild; or a self-organized working group
+that a team member lead a particular guild; or a self-organized [working group](working-group)
```
Comment (mbland) [inline pages/guilds.md]: `(working-group)` => ({{site.baseurl}}/working-groups/)`

File: pages/guilds.md [hunk 3]
```diff
@@ -51,33 +53,32 @@ work, and make a formal announcement if the guild is created.
...
-ensure a guild is setting objectives and holding itself accountable.
+ensure a guild is holding itself accountable to its goals.
 
 Guild sponsors will convene one meeting each quarter to evaluate guild
 progress. Based on the outcome of this meeting, the sponsor may recommend
 spinning down a guild or revoke sponsorship altogether. If a guild has its
-sponsorship revoked, it may continue to operate as a working group, but will
-no longer receive mentorship and advocacy from a sponsor.
+sponsorship revoked, its charter may need to be revised to remove project team
+requirements and may lose dedicated resources, however, it may continue to operate as a working group.
```
Comment (mbland) [inline pages/guilds.md]: Nit: semicolon before "however".

File: pages/guilds.md [hunk 4]
```diff
@@ -51,33 +53,32 @@ work, and make a formal announcement if the guild is created.
...
-in service to organizational priorities.
+the objectives of the broader team than working groups. Collaboration with
+the guild sponsor allows for aligning guild objectives with 18F organizational priorities.
 
-Broadly speaking, guild responsibilities include:
+Guild responsibilities include:
 
-* Establishing [standards and/or best practices](http://18f.github.io/guides/)
+* Establish a Charter
+* Develop [standards and/or best practices](http://18f.github.io/guides/)
```
Comment (mbland) [inline pages/guilds.md]: The link can now be `https://pages.18.gov/guides/`.

File: pages/guilds.md [hunk 5]
```diff
@@ -51,33 +53,32 @@ work, and make a formal announcement if the guild is created.
...
-representation is a responsibility for setting and meeting objectives that are
-in service to organizational priorities.
+the objectives of the broader team than working groups. Collaboration with
+the guild sponsor allows for aligning guild objectives with 18F organizational priorities.
 
-Broadly speaking, guild responsibilities include:
+Guild responsibilities include:
 
-* Establishing [standards and/or best practices](http://18f.github.io/guides/)
+* Establish a Charter
```
Comment (mbland) [inline pages/guilds.md:73]: Nit: Reads better to me if these start with gerunds (e.g. "Establishing"), but not worth changing unless you agree.

**Comment by mbland** [comment]
Tiny comments aside, this looks great to me!

File: pages/guilds.md (hunk 5 above)
Comment (ultrasaurus) [inline pages/guilds.md:73]: I like shorter.  Imperative feels like there's a sense of urgency :)

**Comment by ultrasaurus** [comment]
thanks so much for the feedback @mbland -- will submit an update addressing almost all your comments :)

**Comment by gbinal** [comment]
+1 to merging this, driving 18F leadership to affirm or suggest changes, and then to roll out to the wider team

**Comment by gbinal** [comment]
p.s. Thanks for this, @ultrasaurus

**Comment by ultrasaurus** [comment]
agree with @gbinal's suggestion to merge -- if other folks have ideas on improvements, feel free to file issues or propose changes via pull requests

on guild sponsorship, we are arguably sponsored since we were eached ask to take on these roles.  We all have managers or can approach @aaronsnow or @quepol directly for now.  I like the process improvement to have specific individual sponsors, but that's not elaborated in detail in the doc (yet).

**Comment by quepol** [comment]
Sorry for the delay. I made a couple of minor typo/grammar edits, but the doc looks great.

Regarding sponsorship, I'll put it on the agenda for the next MGMT meeting. Crucial that we formalize that part of it ASAP, especially as we head into OKR-land as an organization.

I have one suggestion for the Privileges section, but I'll do a PR so we can discuss it separately there. Great working pulling this together!

**Pull request merged**